app/controllers/concerns/authentication_and_sso_concerns.rb

```diff
@@ -4,16 +4,28 @@
 # to this responsibility alone.
 module AuthenticationAndSSOConcerns
   extend ActiveSupport::Concern
-  include ActionController::HttpAuthentication::Token::ControllerMethods
```

Comment:
Don't we still need `ActionController::HttpAuthentication::Token::ControllerMethods` to do token authentication?

Reply:
This should probably be in a separate PR. 
This should have been removed in PR #2692. None of the methods it provides are in use anymore.

Reply:
Those methods documented [here](https://api.rubyonrails.org/v5.2.3/classes/ActionController/HttpAuthentication/Basic/ControllerMethods.html)